Update banner styling for Forge 6.7

DFurnes opened the pull request:
#### Changes

Some fresh new CSS! This PR adds the new banner style that's going to be introduced in Forge 6.7 to the pattern library, replacing the old banner style. :art: 
#### Screenshots

With shorter heading text:
<img width="471" alt="screen shot 2016-03-21 at 3 51 13 pm" src="https://cloud.githubusercontent.com/assets/583202/13932051/c51e503c-ef7c-11e5-95ed-b9c55e861124.png">

With longer text that expands past the yellow bar:
<img width="643" alt="screen shot 2016-03-21 at 3 38 27 pm" src="https://cloud.githubusercontent.com/assets/583202/13931894/e0a4ba5e-ef7b-11e5-8801-b5106d535c4f.png">

With extra-long text that wraps to another line:
<img width="612" alt="screen shot 2016-03-21 at 3 39 07 pm" src="https://cloud.githubusercontent.com/assets/583202/13931914/f390bb68-ef7b-11e5-8491-639bd6b59b21.png">

And here it is on the profile page:
![screen shot 2016-03-21 at 4 34 33 pm](https://cloud.githubusercontent.com/assets/583202/13933303/de631d42-ef82-11e5-8c3c-cc2dc0dab1df.png)

And on the action page:
![screen shot 2016-03-21 at 4 34 41 pm](https://cloud.githubusercontent.com/assets/583202/13933299/d91ac27c-ef82-11e5-807d-5db027197fc1.png)

And finally, here's how things look on mobile devices:
<img width="288" alt="screen shot 2016-03-21 at 4 42 51 pm" src="https://cloud.githubusercontent.com/assets/583202/13933552/1b171a6c-ef84-11e5-94a1-5f54d7dfb490.png">

---

For review: @DoSomething/front-end @lkpttn

DFurnes committed:
Update banner styling for Forge 6.7

DFurnes commented:
Ignore the :red_circle: from Wercker, that got fixed up in #535.

weerd commented:
Does it make sense to still call this modifier `.-banner`? Just throwing that out there... I like the new look, but doesn't really seem like a banner now. Maybe `.-emphasized`

weerd commented:
Not a huge deal though.. just thought I'd mention it. If otherwise fine, I'm cool with it ;) Would mean updating the templates, which is probs not much fun...

DFurnes commented:
Mmm, I'd be open to adding `.-emphasized` as an alternative class name option (and perhaps deprecating `.-banner`), but since this is replacing the existing banner pattern I don't want to do anything that'd break existing client applications.

lkpttn commented:
+1 C'Thun
+1 Design

We might have to revisit the bottom space after we update the typographic scale..

weerd commented:
@DFurnes yeah, good points. Maybe add a note regarding that somewheres?

:+1:

DFurnes committed:
Add inverse modifier for Heading banner.

DFurnes commented:
@weerd Good idea! Added #537 to track that.

Also just added a `.-banner.-inverse` for the "prove it" section in 0e4fb55.

![screen shot 2016-03-21 at 4 59 47 pm](https://cloud.githubusercontent.com/assets/583202/13934117/c5013eb6-ef86-11e5-8d17-352d44e8a7fd.png)

weerd commented:
:+1: :dancers:
